internal/marketplaces/service.go

```diff
@@ -31,18 +35,21 @@ import (
 // from bundles to projects. Subscriptions are implicitly created and managed
 // by these operations.
 type Marketplace interface {
+	// Subscribe creates a subscription between the specified project and
+	// bundle and adds all rules from that bundle to the project.
 	Subscribe(
 		ctx context.Context,
 		project types.ProjectContext,
 		bundleID mindpak.BundleID,
-		qtx db.ExtendQuerier,
+		qtx db.Querier,
```

Comment:
We do not actually need `ExtendQuerier` here - use `Querier` for the interfaces which need transactions